themes/default/build/sass/style.scss

```diff
@@ -1,68 +1,96 @@
+5.  UI kit
+6.  Forms
+7.  Components
+8.  Shame
+------------------------------------------------------------------------- 
+*/
+
+/* 
+-------------------------------------------------------------------------
+1.  Pure
+	Import pure as sass for extending
+------------------------------------------------------------------------- 
+*/
 
-Define the colours and fonts used throughout the web app
+@import '../../css/pure.src';
 
-Styleguide 1.0
+/*
+-------------------------------------------------------------------------
+2.  Brand overview
+	Define the colours and fonts used throughout the web app
+------------------------------------------------------------------------- 
 */
+
 @import "utilities/functions";
 @import "utilities/var-globals";
 @import "utilities/respond";
 @import "utilities/classes";
 
 /*
-# Layout
-
-Grid and layout related styling. We are using pure's grid system. More in depth documentation can be found at:
-http://purecss.io/grids/
-
-Styleguide 2.0
+-------------------------------------------------------------------------
+3.  Layout
+	Grid and layout related styling. We are using pure's grid system. 
+	More in depth documentation can be found at: http://purecss.io/grids
+------------------------------------------------------------------------- 
 */
+
 @import "layout/*";
 
 /*
-# Typography
-
-Styles for typographical and related elements editable in Silverstripe's HTML Text editor.
-This includes headings, paragraphs, quotes, links, lists, images, media, and tables.
-
-Styleguide 3.0
+-------------------------------------------------------------------------
+4.  Typography
+    Styles for typographical and related elements editable in 
+    Silverstripe's HTML Text editor. This includes headings, paragraphs, 
+    quotes, links, lists, images, media, and tables.
+------------------------------------------------------------------------- 
 */
+
 @import "typography/*";
 
 /*
-# UI Kit
-
-A place for the small elements, such as buttons, that make up larger components
-
-Styleguide 4.0
+-------------------------------------------------------------------------
+5.  UI Kit
+    A place for the small elements, such as buttons, that make up larger 
+    components
+------------------------------------------------------------------------- 
 */
+
 @import "ui-kit/*";
 
 /*
-# Forms
-
-Form, and form element styles
-
-Styleguide 5.0
+-------------------------------------------------------------------------
+6.  Forms
+    Form, and form element styles
+------------------------------------------------------------------------- 
 */
 
-
 @import "forms/*";
 
 /*
-# Components
-
-Reusable and customizable front-end UI elements
-
-Styleguide 6.0
-
+-------------------------------------------------------------------------
+7.  Components
+    Reusable and customizable front-end UI elements
+------------------------------------------------------------------------- 
 */
-@import "components/**/*";
 
+@import "components/**/*";
 
+/*
```

Comment:
We probably don't want this one in our styleguide... feel free to // single line comment the file import though.